dev-docs/modules/s2sTesting.md

```diff
@@ -4,6 +4,8 @@ title: Module - Prebid Server Testing
 description: Adds A/B test support to ease analysis of server-side header bidding
 top_nav_section: dev_docs
 nav_section: modules
+module_code : s2sTesting
+display_name : Prebid Server Testing
```

Comment:
I'm running this branch locally, and I'm seeing this referred to as:

- s2sTesting
- Prebid Server Testing
- Server-to-Server Testing

I think we should standardize to call it "Server-to-Server Testing" anywhere it appears in English text (such as the page title and on the Downloads page).